README.rst

```diff
@@ -205,13 +205,26 @@ You must have `Docker installed <https://docs.docker.com/engine/installation/>`_
 
         docker run -d --name quipucords -e "QPC_DBMS=sqlite" -p443:443 -i quipucords:1.0.0
 
+    D. For debugging purposes you may want to run the Docker image with the /app directory mapped to your local clone of quipucords and the logs mapped to a temporary directory::
```

Comment:
I would clarify the purpose a bit.  Viewing the logs isn't that difficult.  Rebuilding the container is painful.  Clarify that the purpose is to allow you to rapidly change server code without rebuilding the container.